sdk/eventgrid/azure-messaging-eventgrid-perf/src/main/java/com/azure/messaging/eventgrid/perf/SendEventGridEventsTest.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventgrid.perf;
+
+import com.azure.messaging.eventgrid.EventGridEvent;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SendEventGridEventsTest extends ServiceTest<PerfStressOptions> {
+
+    public SendEventGridEventsTest(PerfStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public void run() {
+        eventGridEventPublisherClient.sendEventGridEvents(createEvents());
```

Comment:
The create Events method needs to be moved to setupAsync method or the constructor, store the events as a local Array List.

Reply:
I prefer calling createEvents for every send because it's more like user scenario, which send out different events for every call.